src/main/resources/application-scone.yml

```diff
@@ -31,3 +31,5 @@ tee:
     attestation:
       tolerated-insecure-options: ${IEXEC_SCONE_TOLERATED_INSECURE_OPTIONS:} # e.g.: hyperthreading,software-hardening-needed,insecure-igpu,outdated-tcb,debug-mode
       ignored-sgx-advisories: ${IEXEC_IGNORED_SGX_ADVISORIES:} # e.g.: INTEL-SA-00220,INTEL-SA-00270
+      mode: ${IEXEC_SCONE_ATTESTATION_MODE:hardware} # hardware or maa
+      url: ${IEXEC_SCONE_ATTESTATION_URL:#{null}} # required when mode is maa
```

Comment:
Sadly, this can not work.
See integration tests last run to see what happens (see failing SMS in enclave tests).

Reply:
Is it only the `#{null}` string that should just be `null` or the use of environment variable that is not a viable way to do the thing ?

Reply:
As Scone waits for a valid URL here, no value seems to really work here while `hardware` is the default mode.
You can take some time to explore things here, on my local env, I had to comment this line and remove the idea of default value here.

Reply:
default values modified to `maa` mode with non-null URL